src/WinRT.Runtime2/InteropServices/ExceptionErrorInfo.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+using WindowsRuntime.InteropServices;
+using WindowsRuntime.InteropServices.Marshalling;
+
+namespace ABI.WinRT.Interop;
+
+internal static class IRestrictedErrorInfoMethods
```

Comment:
Code structure:
- We should have one file per `*Methods` class
- Let's put these under `InteropServices\Exceptions` (that folder doesn't currently exist)
- Namespace should be `WindowsRuntime.InteropServices`

Reply:
Would it make sense to just combine all these static classes into one?